docs/installation/manual/installing_fedora_syn_and_blazegraph.md

```diff
@@ -50,194 +47,97 @@ grant all privileges on database FEDORA_DB to FEDORA_DB_USER;
-        init-method="init" destroy-method="shutdown">
-        <property name="configPath" value="${fcrepo.external.content.allowed:#{null}}" />
-        <property name="monitorForChanges" value="true" />
-    </bean>
-    <bean name="externalContentHandlerFactory" class="org.fcrepo.http.api.ExternalContentHandlerFactory">
-        <property name="validator" ref="externalContentPathValidator" />
-    </bean>
-    <task:scheduler id="taskScheduler" />
-    <task:executor id="taskExecutor" pool-size="1" />
-    <task:annotation-driven executor="taskExecutor" scheduler="taskScheduler" />
-    <bean class="org.modeshape.jcr.ModeShapeEngine" init-method="start"/>
-    <bean id="connectionManager" class="org.apache.http.impl.conn.PoolingHttpClientConnectionManager" />
-    <bean class="org.fcrepo.http.commons.session.SessionFactory"/>
-</beans>
+**Note**: the trailing backslash is important here. For more information on Fedora's External Content and configuring it, see the [Fedora Wiki pages](https://wiki.lyrasis.org/display/FEDORA6x/External+Content)
+
+#### Fedora configuration properties file
+
+Fedora 6 now allows you to put all your configuration properties into a single file. We use `0640` permissions as you will want to put your database credentials in here.
+
+We've left `{{ fcrepo X property }}` to define configurable properties
+
+`/opt/fcrepo/config/fcrepo.properties | tomcat:tomcat/640`
+```{ .text .copy }
+fcrepo.home=/opt/fcrepo/data
+fcrepo.external.content.allowed=/opt/fcrepo/config/allowed_external_hosts.txt  # Default using path defined above.
+fcrepo.namespace.registry=/opt/fcrepo/config/i8_namespaces.yml  # Default using path defined above.
+fcrepo.auth.principal.header.enabled=true
+fcrepo.auth.principal.header.name=X-Islandora  # The same as the syn-setting.xml "config" element's "header" attribute
+fcrepo.autoversioning.enabled={{ fcrepo_autoversioning }}  # false to use manual versioning, true to create a version on each change
+fcrepo.db.url=jdbc:{{ fcrepo_db_type }}://{{ fcrepo_db_host }}:{{ fcrepo_db_port }}/{{ fcrepo_db_name }}
+fcrepo.db.user={{ fcrepo_db_user }}
+fcrepo.db.password={{ fcrepo_db_password }}
+fcrepo.ocfl.root={{ fcrepo_data_dir }}/ocfl-root  # Default is {{ fcrepo.home }}/ocfl-root, but can be where you want your data
+fcrepo.ocfl.temp={{ fcrepo_data_dir }}/temp  # Same as above, temp OCFL space.
+fcrepo.ocfl.staging={{ fcrepo_data_dir }}/staging  # Same as above, staging OCFL space for transactions. 
+fcrepo.namespace.registry=/opt/fcrepo/config/namespaces.yml  # Default based on above path.
+fcrepo.persistence.defaultDigestAlgorithm=sha512  # Can be sha512 or sha256
+fcrepo.dynamic.jms.port=61626  # Moved from 61616 to allow external ActiveMQ to use that port
+fcrepo.dynamic.stomp.port=61623  # Same as above
+fcrepo.velocity.runtime.log=/opt/tomcat/logs/velocity.log
+fcrepo.jms.baseUrl=http://localhost:8080/fcrepo/rest  # Base URL set on emitted JMS messages
 ```
```

Comment:
This file seems to break convention with the rest of this manual, which was using ALL_CAPS for variables then under the file block, defining what they should be. Would you mind doing that? In particular I'm not sure what to use for `{{ fcrepo_data_dir }}` or even `{{ fcrepo.home }}` based on the above instructions (probably `/opt/fcrepo` or /opt/fcrepo/data/objects`?)

Reply:
Just to make sure, you want me to change the variables from `{{ fcrepo_db_user }}` to `FCREPO_DB_USER`.

I can also try to add more information as well but I do think that if people want to customize their Fedora setup the information on the link below is clearer than I can do. 

All of this is customizable so maybe it's easier to just add some values and let people change it if they want 🤷

Reply:
For example

```{ .text .copy }
fcrepo.home=/opt/fcrepo/data
fcrepo.external.content.allowed=/opt/fcrepo/config/allowed_external_hosts.txt  # Default using path defined above.
fcrepo.namespace.registry=/opt/fcrepo/config/i8_namespaces.yml  # Default using path defined above.
fcrepo.auth.principal.header.enabled=true
fcrepo.auth.principal.header.name=X-Islandora  # The same as the syn-setting.xml "config" element's "header" attribute
fcrepo.autoversioning.enabled=true  # false to use manual versioning, true to create a version on each change
fcrepo.db.url=jdbc:mysql://localhost:3306/fedora_database
fcrepo.db.user=fedora_user
fcrepo.db.password=fedora_user_password
fcrepo.ocfl.root=/opt/fcrepo/data/ocfl-root  # Default is {{ fcrepo.home }}/ocfl-root, but can be where you want your data
fcrepo.ocfl.temp=/opt/fcrepo/data/temp  # Same as above, temp OCFL space.
fcrepo.ocfl.staging=/opt/fcrepo/data/staging  # Same as above, staging OCFL space for transactions. 
fcrepo.namespace.registry=/opt/fcrepo/config/namespaces.yml  # Default based on above path.
fcrepo.persistence.defaultDigestAlgorithm=sha512  # Can be sha512 or sha256
fcrepo.dynamic.jms.port=61626  # Moved from 61616 to allow external ActiveMQ to use that port
fcrepo.dynamic.stomp.port=61623  # Same as above
fcrepo.velocity.runtime.log=/opt/tomcat/logs/velocity.log
fcrepo.jms.baseUrl=http://localhost:8080/fcrepo/rest  # Base URL set on emitted JMS messages
```

Reply:
This section's [Introduction](https://islandora.github.io/documentation/installation/manual/introduction/) laid out the rules for how things are formatted and they've been followed as far as i can tell, up till now. It works much nicer to have the same paradigm throughout (and to be honest throughout all the documentation), so I feel bad breaking the trend .

I also really don't know what my fcrepo.home should be, so I need the "and-if-you've-been-following-along" value.

Reply:
Ok have a look and let me know if that is better.